packages/@aws-cdk/aws-eks/README.md

```diff
@@ -397,6 +397,46 @@ new KubernetesPatch(this, 'hello-kub-deployment-label', {
 })
 ```
 
+### Querying Kubernetes Resources
+
+The `KubernetesGet` construct can be used to query for information about kubernetes resources,
+and use that as part of your CDK application.
+
+For example, you can fetch the address of a [`LoadBalancer`](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) type service:
+
+```typescript
+// query the load balancer address
+const myServiceAddress = new KubernetesGet(this, 'LoadBalancerAttribute', {
```

Comment:
Should we rename `KubernetesResource` to `KubernetesApply`?

Reply:
Sure

Reply:
Decide against this. See breaking change notice in the PR description.